features/test/storage.feature

```diff
@@ -23,3 +23,10 @@ Feature: some storage related scenarios
   @admin
   Scenario: Store PV into clipboard
     Given admin stores all persistentvolumes to the clipboard
+
+  @admin
+  Scenario: install lso
```

Comment:
perhaps use thee full name instead to be clear (local-storage operator)

Reply:
It's not a test case, just a test code for the new steps. I'll update it.